python/cudf/dataframe/dataframe.py

```diff
@@ -1766,9 +1766,10 @@ def quantile(self, q, interpolation='linear', exact=False):
         result = DataFrame()
         result['Quantile'] = q
         for k, col in self._cols.items():
-            result[k] = col.quantile(q, interpolation, exact,
-                                     quant_index=False)
-        print(result)
+            if columns is None or k in columns:
+                result[k] = col.quantile(q, interpolation, exact,
+                                         quant_index=False)
```

Comment:
If easy then we might want to use keywords here in case Pandas changes their API in the future (which happens)

Reply:
What do you mean by keywords?

Reply:
col.quantile(q, interpolation=interpolation, exact=exact, quant_index=False)

Reply:
Oh 👍